test/verify/check-system-services

```diff
@@ -612,9 +612,24 @@ WantedBy=default.target
 
             # system timers are not allowed to be deleted
             self.goto_service("cockpit-system.timer")
-            b.click(".service-top-panel .pf-v5-c-dropdown button")
-            b.wait_in_text(".service-top-panel .pf-v5-c-dropdown__menu", "Disallow running")
-            b.wait_not_in_text(".service-top-panel .pf-v5-c-dropdown__menu", "Delete")
+            # Check our custom kebab menu in RTL layout separately as the tests switch to RTL layout by editing
+            # html directly while in a real life scenario we would reload the page. This circumvents the PF dropdown
+            # from rendering in the wrong place when making a pixel test.
+            b.set_layout("rtl")
+            # wait for layout to settle
+            time.sleep(1)
```

Comment:
This now sleeps every test, maybe we want to guard this for when we run pixel tests? It would be a bit ugly however.

Reply:
Well, yeah, but I really don't want to block you on such stupid stuff, you've sunken enough time into this :grin: